### ISSUE TITLE
Visual Studio Codespaces will be consolidating into GitHub Codespaces.

### ISSUE THREAD

**Issue by 2percentsilk** [MicrosoftDocs/vscodespaces#788]
We recently announced that Visual Studio Codespaces will be consolidating into GitHub Codespaces.

If you’re an existing Visual Studio Codespaces user, you can start your transition to GitHub Codespaces now and the current Azure offering will be retired in February 2021. [Sign up](https://github.com/features/codespaces/signup) for the GitHub limited public preview.

Check out our [FAQ](https://aka.mcs/vscs/faq/transition) and [announcement](https://aka.ms/vscs-moving) for more information.

**Comment by mschickervxob** [comment]
I gather from the FAQ that it is no longer planned to support self-hosted deployments - which is a setback for all those who bet on it in the last months.

**Comment by timkeeler** [comment]
> I gather from the FAQ that it is no longer planned to support self-hosted deployments - which is a setback for all those who bet on it in the last months.

Oh wow I didn't notice that. Self-hosted was my primary use case and I'm really bummed to see it's no longer planned. I guess I'll just have to rely on remote SSH.

**Comment by appcoders** [comment]
@tiantun 
I am exploring https://github.com/cdr/code-server and https://github.com/gitpod-io/self-hosted now as alternatives.

**Comment by lvmajor** [comment]
I was also really liking the self-hosted option in VS Codespaces, sad to see it go (for now !) :( Now hoping it will eventually be added to Github codespaces

**Comment by MichalCz** [comment]
Yeah, that is a major letdown... Not everyone can just use whatever Github gives you - so this means basically death to VSCode for my project as I need dedicated hardware for it. :(

Back to containers and docker machine manipulation.

**Comment by david50407** [comment]
So sad that self-hosting is not in GitHub Codespace and Microsoft is going to shutdown the old old old VS Codespace before they migrate the user experience to the new one. :(

**Comment by lostintangent** [comment]
Hey all! We've got a feedback item in the GitHub Support Community, that is tracking the discussion around adding self-hosted support to GitHub Codespaces: https://github.community/t/self-hosted-codespaces/131361. We'd love to continue that discussion over there, since that's where we'll start discussing feature requests and roadmap updates. Thanks!

**Comment by mchelen-gov** [comment]
> Check out our [FAQ](https://aka.mcs/vscs/faq/transition) 
Link should be https://aka.ms/vscs/faq/transition

**Comment by srivatsn** [comment]
@mchelen-gov Thanks! Fixed.
